packages/server/middleware/auth.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  checkAdmin: function (req, res, next) {
+    if (process.env.NODE_ENV !== "production") {
+      next();
+    }
```

Comment:
We might want to do a note that this should be removed. Ideally everyone would share a single login that'd be saved as a session/cookie